x-pack/test/cloud_security_posture_functional/pages/findings.ts

```diff
@@ -173,11 +204,29 @@ export default function ({ getPageObjects, getService }: FtrProviderContext) {
           await distributionBar.filterBy(type);
 
           const items = data.filter(({ result }) => result.evaluation === type);
-          expect(await table.getFindingsCount(type)).to.eql(items.length);
+          expect(await latestFindingsTable.getFindingsCount(type)).to.eql(items.length);
 
           await filterBar.removeFilter('result.evaluation');
         });
       });
     });
+
+    describe('Group By', () => {
+      it('Group findings by Resource', async () => {
+        await comboBox.set('findings-group-by-selector', 'Resource');
+        /* Findings by resource table has no Rule column */
```

Comment:
it shouldn't matter what columns another page has, i don't thin we need this comment. 

you could also assert there is at least 1 result and that we've seen the specific page container